src/main/resources/db/migration/common/V2024.12.30.0001__drop_display_name_fields_from_goal_table.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE goal
+    DROP COLUMN created_by_display_name,
+    DROP COLUMN updated_by_display_name;
```

Comment:
Whilst I've updated the `StepEntity` class (above) to be a data class etc, steps have never contained the "display name" fields, so no need to drop any columns etc from `step` table